include/Gaffer/TweakPlug.h

```diff
@@ -70,9 +70,10 @@ class GAFFER_API TweakPlug : public Gaffer::ValuePlug
 			ListAppend,
 			ListPrepend,
 			ListRemove,
+			CreateDefault,
```

Comment:
How are you feeling about the `CreateDefault` name? I know I suggested it, but I'm worried that it's not entirely obvious. Would `CreateIfMissing` be more self-explanatory, and tie better with `MissingMode`?

Reply:
I do like `CreateIfMissing` better. The first time I read `CreateDefault` it took me an extra tick to get what its intention is. Changed in https://github.com/GafferHQ/gaffer/pull/5047/commits/19418b4fa6215cc4afabe83fbd08f1c8acb24563